python/pylibcudf/pylibcudf/interop.pxd

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from pylibcudf.table cimport Table
+
+
+cpdef Table from_dlpack(managed_tensor)
```

Comment:
I agree with Matt, we may as well be explicit with types even when they are `object`.
```suggestion
cpdef Table from_dlpack(object managed_tensor)
```